apps/SageAccountWeb/src/App.tsx

```diff
@@ -141,6 +143,16 @@ function App() {
                               >
                                 <CertificationQuiz />
                               </Route>
+                              <Route
+                                path={'/authenticated/2fa/enroll'}
+                                exact
+                                render={() => <TwoFactorAuthEnrollmentPage />}
+                              />
+                              <Route
+                                path={'/authenticated/2fa/generatecodes'}
+                                exact
+                                render={() => <TwoFactorAuthBackupCodesPage />}
+                              />
```

Comment:
These pages have potentially destructive side-effects. While those side-effects shouldn't occur if the pages are rendered but not shown, I'd rather be extra careful and put them in `render`